src/components/organisms/OcRecipient/OcRecipient.vue

```diff
@@ -123,7 +123,7 @@ export default {
 
 <docs>
```

Comment:
Font of the OcRecipient looks weird

Reply:
Ok sorry not only OcRecipient, but some components seem to be not affected by this